[2021-07-22 15:52:55 -0400] [2168] [INFO] Starting gunicorn 20.1.0
[2021-07-22 15:52:55 -0400] [2168] [INFO] Listening at: http://127.0.0.1:8090 (2168)
[2021-07-22 15:52:55 -0400] [2168] [INFO] Using worker: sync
[2021-07-22 15:52:55 -0400] [2171] [INFO] Booting worker with pid: 2171
[2021-07-22 15:52:55 -0400] [2171] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/dsuazo/flasktest/labelbee/env/lib64/python3.6/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/dsuazo/flasktest/labelbee/env/lib64/python3.6/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/dsuazo/flasktest/labelbee/env/lib64/python3.6/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/dsuazo/flasktest/labelbee/env/lib64/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/dsuazo/flasktest/labelbee/env/lib64/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/dsuazo/flasktest/labelbee/env/lib64/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/dsuazo/flasktest/labelbee/env/lib64/python3.6/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 948, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'manage'
[2021-07-22 15:52:55 -0400] [2171] [INFO] Worker exiting (pid: 2171)
[2021-07-22 15:52:55 -0400] [2168] [INFO] Shutting down: Master
[2021-07-22 15:52:55 -0400] [2168] [INFO] Reason: Worker failed to boot.
